app/posts/manage-your-appointments/2024/09/2024-09-24-what-is-mya.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "What is the manage your appointments service?"
+date: 2024-09-24
+---
+The service will enable healthcare providers to create and manage availability and appointments 
+
+The service is set up to initially support vaccinations, but over a longer term, could be used to support appointments for other services if needed. 
+
+## The background  
+
+A 3rd party appointments management solution was procured by the NHS in response to the Covid-19 pandemic.   
+
+the MYA service was designed to replace the existing 3rd party software, and provide more flexibility for expansion of appointment management services. 
+
+
+## Where we are 
+ 
+We’ve built an MVP, and are testing our service with users delivering the RSV vaccination service in the east of England. 
+
+The service will expand in line with the community pharmacy expansion initiative, and we plan to continue testing and improving the service, to support further expansion, including Covid-19 and flu vaccines. 
```

Comment:
I think [COVID-19](https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#covid-19) is capitalised (but not flu)?

```suggestion
The service will expand in line with the community pharmacy expansion initiative, and we plan to continue testing and improving the service, to support further expansion, including COVID-19 and flu vaccines. 
```